locale/shine/extensions/votealltalk/ruRU.json

```diff
@@ -1,3 +1,14 @@
 {
 	"NOTIFY_PREFIX": "[Межкомандное общение]"
+	"PLAYER_VOTED_ENABLE": "{PlayerName} выбрал включение межкомандного общения. Необходимо ещё {VotesNeeded}.",
+	"PLAYER_VOTED_DISABLE": "{PlayerName} выбрал отключение межкомандного общения. Необходимо ещё {VotesNeeded}.",
+	"PLAYER_VOTED_PRIVATE_ENABLE": "Вы проголосовали за включение межкомандного общения. Необходимо ещё {VotesNeeded}.",
+	"PLAYER_VOTED_PRIVATE_DISABLE": "Вы проголосовали за отключение межкомандного общения. Необходимо ещё {VotesNeeded}.",
+	"ERROR_ALREADY_VOTED_ENABLE": "Вы уже проголосовали за включение межкомандного общения.",
+	"ERROR_ALREADY_VOTED_DISABLE": "Вы уже проголосовали за отключение межкомандного общения.",
+	"ERROR_MUST_WAIT_ENABLE": "Вы должны подождать {SecondsToWait:Duration} сек. перед голосованием за включение межкомандного общения.",
+	"ERROR_MUST_WAIT_DISABLE": "Вы должны подождать {SecondsToWait:Duration} сек. перед голосованием за отключение межкомандного общения.",
+	"ERROR_CANNOT_VOTE_IN_SPECTATE": "Наблюдателям не разрешено голосовать за включение межкомандного общения.",
+	"DISABLE_ALLTALK": "Отключить межкомандное общение",
+	"ENABLE_ALLTALK": "Включить межкомандное общение"
```

Comment:
These last 2 strings are used on the vote menu button, and end up quite a bit larger than the button is. Is there a shorter way of writing this? It's not a huge problem if not, just means the text overflows a bit.

Reply:
Will fix asap